internal/cmd/server/add_to_placement_group.go

```diff
@@ -41,7 +41,7 @@ var AddToPlacementGroupCmd = base.Cmd{
 			return err
 		}
 		if placementGroup == nil {
-			return fmt.Errorf("placement group not found %s", placementGroupIDOrName)
+			return fmt.Errorf("Placement Group not found: %s", placementGroupIDOrName)
```

Comment:
The `:` is new, any reason to deviate from the pattern also used for the server not found error?

Should we use a helper function to make not found errors and use it across the project?